junifer/storage/pandas_base.py

```diff
@@ -30,14 +30,13 @@ class PandasBaseFeatureStorage(BaseFeatureStorage):
 
     See Also
     --------
-    BaseFeatureStorage
+    BaseFeatureStorage : The base class for feature storage.
```

Comment:
The format is inconsistent with the previous instances, which one do we go for?

Reply:
this is numpydoc, I would go that format.